src/main/java/graphql/ExecutionInput.java

```diff
@@ -14,14 +18,17 @@
     private final Object context;
     private final Object root;
     private final Map<String, Object> variables;
+    private final DataLoaderRegistry dataLoaderRegistry;
 
 
-    public ExecutionInput(String query, String operationName, Object context, Object root, Map<String, Object> variables) {
```

Comment:
I might want to change this to prevent broken API for those who called create direct